core/src/main/scala/org/genivi/sota/core/rvi/TransferController.scala

```diff
@@ -125,6 +129,11 @@ class TransferProtocolActor(db: Database, rviClient: RviClient, transferActorPro
         }
       }
 
+    case UploadAborted =>
```

Comment:
The server aborts the process. From its perspective it is upload, as it is initiated by the server, kind of push. So I believe the naming is clear. But you are kind of native speaker, so it is your call ;) Should i rename it?